config/locales/en.yml

```diff
@@ -6296,6 +6296,7 @@ en:
   view_more_observations_of_x_project: "View more observations from %{project} on <nobr>%{site_name} &raquo;</nobr>"
   view_more_observations_of_x_taxon: "View more observations of %{taxon} on <nobr>%{site_name} &raquo;</nobr>"
   view_more_unreviewed: "View more unreviewed"
+  view_my_affected_content: View my affected content
```

Comment:
Let's change this to "View my affected identifications" since the tool being linked to doesn't show *all* content affected by the change, just the identifications.

Also, please add a comment explaining to translators what this means, e.g. "Link to view identifications automatically added by a taxon change"